packages/list-reusable-blocks/src/components/import-dropdown/index.js

```diff
@@ -20,11 +20,7 @@ function ImportDropdown( { onUpload } ) {
 			position="bottom right"
 			contentClassName="list-reusable-blocks-import-dropdown__content"
 			renderToggle={ ( { isOpen, onToggle } ) => (
-				<Button
-					aria-expanded={ isOpen }
-					onClick={ onToggle }
-					isPrimary
-				>
+				<Button aria-expanded={ isOpen } onClick={ onToggle } isPrimary>
```

Comment:
I don't understand why this change is here?

Reply:
Me either!